Documentation/gettingstarted.rst

```diff
@@ -220,7 +220,7 @@ We can achieve that with the following Kubernetes NetworkPolicy:
         - podSelector:
             matchLabels:
               id: app2
-        ports:
```

Comment:
Are you undoing all of these later on again? I did this commit to stop showing L3/L4 combined policies in the guides when we don't support them. We never merged it though. Given that we now support it, this commit is no longer required. So we can either just skip it here and rebase or undo it at the end.

Reply:
Let me look at skipping it here and see whether anything falls out.